docker-compose.yml

```diff
@@ -1,5 +1,34 @@
 version: '3.8'
 services:
+  dre:
+    build: .
+    environment:
+      APPSYNC_KEY: "${APPSYNC_KEY}"
+      NODE_JWK_KEY: "${NODE_JWK_KEY}"
+      STREAMR_STREAM_ID: "${STREAMR_STREAM_ID}"
+      PUBSUB_TYPE: "${PUBSUB_TYPE:-streamr}"
+      GW_PORT: "${GW_PORT}"
+      GW_HOST: "${GW_HOST}"
+      GW_USERNAME: "${GW_USERNAME}"
+      GW_PASSWORD: "${GW_PASSWORD}"
+      GW_TLS: "${GW_TLS}"
+      GW_ENABLE_OFFLINE_QUEUE: "${GW_ENABLE_OFFLINE_QUEUE:-true}"
+      GW_LAZY_CONNECT: "${GW_LAZY_CONNECT}"
+      BULLMQ_PORT: "${BULLMQ_PORT}"
+      BULLMQ_HOST: "${BULLMQ_HOST:-cache}"
+      BULLMQ_USERNAME: "${BULLMQ_USERNAME}"
+      BULLMQ_PASSWORD: "${BULLMQ_PASSWORD}"
+      BULLMQ_TLS: "${BULLMQ_TLS}"
+      BULLMQ_ENABLE_OFFLINE_QUEUE: "${BULLMQ_ENABLE_OFFLINE_QUEUE}"
+      BULLMQ_LAZY_CONNECT: "${BULLMQ_LAZY_CONNECT}"
+      WORKERS_REGISTER: "${WORKERS_REGISTER}"
+      WORKERS_UPDATE: "${WORKERS_UPDATE}"
+      WORKERS_JOB_ID_REFRESH_SECONDS: "${WORKERS_JOB_ID_REFRESH_SECONDS}"
+      WORKERS_MAX_FAILURES: "${WORKERS_MAX_FAILURES}"
+
+    volumes:
+      - dre-sqlite:/app/sqlite
```

Comment:
I've created two volumes - for sqlite and cache, mapped in the different folders. Any reasons you see it shouldn't work? I think I didn't get the point. :)

Reply:
Oh, it won't be "local directory mount". It's a "local" docker driver, which creates local volume inside the docker. With this configuration, it will create a named volume and it will reuse the volume on container restart. And it shouldn't be a host directory here. You can be connected to remote docker with custom configuration, running without permissions to host filesystem.

Reply:
ok, without getting too much into docker stuff - it is crucial to be able to easily access the cache files from the host.
How it will be possible in this version?

Reply:
Also, I wonder if the wallet shouldn't be passed as in https://github.com/Bundlr-Network/validator/blob/master/docker-compose.yml#L51 (instead of passing the json in the env file..)

Reply:
I need to understand the problem to make a good solution without data corruption.
I see the possible reasons you could need it:

- debugging proposes 
- some external processes should access it
- another container should access it

Depending on the case there are different solutions for it.  You said you want people to be able to run this not only for debugging proposes but for "easy run" production nodes. If so, we need to think not only about a single happy path in a local environment. If it is the local environment only for us, it's a different case. :)